util/path.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"github.com/mitchellh/go-homedir"
+)
+
+// ExpandPath converts the file path to include the home directory when prefixed with `~`.
+func ExpandPath(path string) (string, error) {
+	return homedir.Expand(path)
+}
```

Comment:
Ok, so I checked and I don't think we need to import this, less baggage. 

The one thing that `homedir.Exapand()` seems to support is expanding path with tilde (`~/go`). Could we achieve the same thing with something like this:
```go
func ExpandPath(path string)(string,error) {
    home, _ := os.UserHomeDir()
    if path[0] == '~' {
        filepath.Join(home, path[1:])
    }
    return path
}
```

Mitchell's code is doing something similar. It only fallsback to other methods if the user home env is not set. You can see Go is already taking care of something similar
https://golang.org/src/os/file.go?s=14399:14433#L479 

Let me know what you think

Reply:
Initially, I tried to do this thing. Then I wasn't sure if that would work or not always, hence used the library instead after some searching. I will revert this, and use the simpler implementation like you have mentioned above.